### PR TITLE
Modified the horse stable form to include owner

### PR DESCRIPTION
**Pull request by Femchengdu**
### Overview:

* Modified the HorseStable form to include user details.
* Created helper method for HorseStable and Booking view.
* Deleted old listing helper file.

### CODE REVIEW

**Comment by Femchengdu** [comment]
It took a while to test the application for bugs after making changes, I was wondering if it is possible to continue development using the test first approach.

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -26,18 +26,19 @@ def show
...
   def create
-	@stall = Stall.find(params[:booking][:stall_id])
-	@booking = @stall.bookings.build
-	@booking.save
-	if @booking.update(booking_params)
-		redirect_to(@booking)
-	else
-		render :new
-	end
+  	@stall = Stall.find(params[:booking][:stall_id])
```
Comment (bodrovis) [inline app/controllers/bookings_controller.rb]: Note that if the record is not found, an error will be raised. Compare this to `find_by`. This is not a mistake though, just remember to rescue the error somewhere.

File: app/views/horse_stables/_form.html.erb [hunk 2]
```diff
@@ -1,5 +1,5 @@
-<%= form_for horse_stable, html: { multipart: true } do |f| %>
-  <%= render 'shared/errors', object: horse_stable %>
+<%= form_for @horse_stable, html: { multipart: true } do |f| %>
```
Comment (bodrovis) [inline app/views/horse_stables/_form.html.erb]: I think `horse_stable` is set when rendering the form. https://github.com/Femchengdu/horse_hotel/blob/master/app/views/bookings/new.html.erb#L3 This way we isolate an object, shall we say. So no need to use an instance variable that may or may not be defined.

File: app/views/horse_stables/_form.html.erb [hunk 3]
```diff
@@ -18,6 +18,10 @@
     <%= f.file_field :image, class: "form-control" %>
   </div>
   <div class="form-group">
+  <%= f.label :owner %>
+  <%= f.select(:user_id, options_for_select(users_for_select)) %>
```
Comment (bodrovis) [inline app/views/horse_stables/_form.html.erb]: I think we can use this method http://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-collection_select

File: app/views/horse_stables/_form.html.erb (hunk 2 above)
Comment (Femchengdu) [inline app/views/horse_stables/_form.html.erb]: I get the following error: First argument in form cannot contain nil or be empty if I don't use the @horse_stable variable.

Comment (bodrovis) [inline app/views/horse_stables/_form.html.erb]: Interesting. I'll check it, probably did not notice something.

**Pull request merged**